app/notifications/validators.py

```diff
@@ -594,3 +595,18 @@ def decode_personalisation_files(json_personalisation):
                 }
             )
     return json_personalisation, errors
+
+
+def validate_notification_does_not_exceed_sqs_limit(notification):
+    # SQS max payload size is 256KB
+    if len(str(notification)) >= 262144:
```

Comment:
Should we make this payload size an environment variable in the event that we perform this type of check elsewhere in the code and/or the SQS size limit changes?

Reply:
yes please